.github/workflows/dep-summaries.yml

```diff
@@ -90,19 +91,24 @@ jobs:
           output="${output//$'\n'/'%0A'}"
           output="${output//$'\r'/'%0D'}"
           echo "::set-output name=diff::${output}"
-
       - name: read dep-auditors list
         id: dep-auditors
-        if: ${{ steps.verify-lsr.outputs.diff || steps.verify-lec.outputs.diff }}
         run: |
           output=$(cat ./.github/dep-auditors.json | jq -r 'join(",")')
           echo "::set-output name=list::${output}"
+
+  requires-dep-audit-review:
+    runs-on: ubuntu-latest
+    name: fail if dep audit review required.
```

Comment:
Place change this field to "Require review for TCB dep summary changes" so that it is more consistent with the other top level names.